web/src/OverviewLogPane.tsx

```diff
@@ -85,6 +85,14 @@ function newLineEl(
   if (buildEvent === "init") {
     classes.push("is-buildEvent")
     classes.push("is-buildEvent-init")
+
+    if (showManifestPrefix) {
+      text += ` • ${line.manifestName}`
```

Comment:
(nit) it'd be nice to have a short comment here that explains the reason for suffixing with the manifest name.

Reply:
done